applications/common/backend/src/main/resources/libs/kyanite/common/components/contactform/contactform.html

```diff
@@ -16,105 +16,140 @@
       </div>
-      <p class="help is-danger is-hidden email-validate-error">This email is invalid</p>
-    </div>
 
-    <div class="field">
-      <div class="control">
-        <input class="input" type="text" name="companyName" placeholder="Company Name">
+      <div class="field">
+        <div class="control">
+          <input class="input" type="text" name="companyName" placeholder="Company Name">
+        </div>
       </div>
-    </div>
 
-    <div  data-sly-attribute.hidden="${!properties.showMessage}"
-          class="field">
-      <div class="control">
-        <textarea class="textarea" name="message" placeholder="${model.messagePlaceholder}"></textarea>
+      <div data-sly-attribute.hidden="${!model.showMessage}"
+           class="field">
+        <div class="control">
+          <textarea class="textarea" name="message"
+                    placeholder="${model.messagePlaceholder}"></textarea>
+        </div>
       </div>
-    </div>
 
-    <div class="field">
-      <div class="control">
-        <label class="checkbox is-flex">
-          <input type="checkbox" class="mr-2" required>
-          ${model.consentText @ context='html'}
-        </label>
+      <div class="field">
+        <div class="control">
+          <label class="checkbox is-flex">
+            <input type="checkbox" class="mr-2" required>
+            ${model.consentText @ context='html'}
+          </label>
+        </div>
       </div>
-    </div>
 
-    <div hidden class="field">
-      <div class="control">
-        <input class="input" type="text" name="pageSpace" value="${model.spaceName}">
+      <div hidden class="field">
+        <div class="control">
+          <input class="input" type="text" name="pageSpace" value="${model.spaceName}">
+        </div>
       </div>
-    </div>
 
-    <div hidden class="field">
-      <div class="control">
-        <input class="input" type="text" name="pageName" value="${model.pageName}">
+      <div hidden class="field">
+        <div class="control">
+          <input class="input" type="text" name="pageName" value="${model.pageName}">
+        </div>
       </div>
-    </div>
 
-    <div class="field">
-      <div class="control">
-        <input id="submit-btn" class="button is-primary is-medium" type="submit"
-               value="${model.submitLabel}" data-form-id="${resource.name}">
-      </div>
-    </div>
-    <div class="block">
-      <div class="notification is-danger is-hidden form-failure mb-0">
-        Something's wrong, try again
+      <div class="field">
+        <div class="control">
+          <input id="submit-btn" class="button is-primary is-medium" type="submit"
+                 value="${model.submitLabel}" data-form-id="${resource.name}">
+        </div>
       </div>
-      <div class="notification is-success is-hidden form-success">
-        Form submitted successfully
+      <div class="block submit-message-type submit-message-default">
```

Comment:
Maybe we could wrap the old message block with `data-sly-test="${!model.showAlternativeSubmitMessage}"` to not produce this block on pages with alternative message?